spec/features/admin/organizations_spec.rb

```diff
@@ -644,7 +644,7 @@
 
           scenario 'Try update organization with invalid legal representant and display error' do
             organization = create(:organization)
-            legal_representant = create(:legal_representant, organization: organization)
+            _legal_representant = create(:legal_representant, organization: organization)
```

Comment:
I guess this `_` prefix comes from rubocop autocorrections, and from my point of view if a variable is not being used, is better to avoid using a variable at all rather than renaming it. Same for the rest of the file

Reply:
As you said, its from rubocop. I'll delete it